python/paddle/fluid/contrib/slim/tests/CMakeLists.txt

```diff
@@ -1,11 +1,59 @@
 file(GLOB TEST_OPS RELATIVE "${CMAKE_CURRENT_SOURCE_DIR}" "test_*.py")
 string(REPLACE ".py" "" TEST_OPS "${TEST_OPS}")
 
+function(inference_analysis_python_api_int8_test target model_dir data_dir filename)
+    py_test(${target} SRCS ${filename}
+        ENVS FLAGS_OMP_NUM_THREADS=${CPU_NUM_THREADS_ON_CI}
+        ARGS --infer_model ${model_dir}/model
+             --infer_data ${data_dir}/data.bin
+             --int8_model_save_path int8_models/${target}
+             --warmup_batch_size 100
```

Comment:
I see you change it to 10 in https://github.com/PaddlePaddle/Paddle/pull/17634/commits/085642a9fdba219180e2084a8ef5bf2d117b207f#diff-1d840481eb99c8cb8ae2116875d02d57R9. Why do you change back to 100 again?

Reply:
Got it. But are 10 images too less to ensure the accuracy?